libs/ledger-live-common/src/errors.ts

```diff
@@ -211,9 +211,9 @@ export const NotImplementedError = createCustomErrorClass(
 export const ValidationError = createCustomErrorClass("ValidationError");
 export const AccessDeniedError = createCustomErrorClass("AccessDeniedError");
 
-export const AlgorandASANotOptInInRecipient = createCustomErrorClass(
-  "AlgorandASANotOptInInRecipient"
-);
+// export const AlgorandASANotOptInInRecipient = createCustomErrorClass(
```

Comment:
You can delete those lines.

Reply:
Done here 69fa69ec14fa367a4245ad0aed44226137fb65ea